src/Sarif.FunctionalTests/ConverterTestData/CppCheck/CppCheck_issueLog1_raw.xml.sarif

```diff
@@ -1163,24 +1163,26 @@
             }
           ],
           "codeFlows": [
-            [
-              {
-                "physicalLocation": {
-                  "uri": "zutil.c",
-                  "region": {
-                    "startLine": 309
+            {
+              "locations": [
+                {
+                  "physicalLocation": {
+                    "uri": "zutil.c",
+                    "region": {
+                      "startLine": 309
+                    }
                   }
-                }
-              },
-              {
-                "physicalLocation": {
-                  "uri": "zutil.c",
-                  "region": {
-                    "startLine": 309
+                },
+                {
+                  "physicalLocation": {
+                    "uri": "zutil.c",
+                    "region": {
+                      "startLine": 309
+                    }
```

Comment:
That's an odd choice of highlighting, but the change is as I showed you: starting with the curly instead of the bracket, and then the `locations` property, on lines 1166-67.